ruby/vampires.rb

```diff
@@ -2,76 +2,100 @@
 puts "How many employees will be processed this session?"
 candidates = gets.chomp
 candidates = candidates.to_i
-# Recieve applicant name.
-puts "Welcome to Werewolf Inc.  Please state your name."
-name = gets.chomp
-# Recieve applicant stated age.
-puts "Please state your age."
-stated_age = gets.chomp.to_i
-# Recieve applicant stated birthday
-puts "Please state your year of birth."
-birth_year = gets.chomp.to_i
-birth_year_age = 2016 - birth_year
-# Age honest is a boolean variable testing if state birth year corresponds with state age.
-age_honesty = (stated_age == birth_year_age)
-# Test applicant garlic preference. Support invalid input.
-valid_input = false
-until valid_input
-	puts "Would you like garlic bread with your lunch? y/n?"
-	garlic = gets.chomp
-	garlic = garlic.downcase
-	if garlic == "y"
-		garlic = true
-		valid_input = true
-	elsif garlic == "n"
-		garlic = false
-		valid_input = true
-	else
-		puts ""
-		puts "I didn't understand you."
+while candidates > 0
+	# Recieve applicant name.
+	puts "Welcome to Werewolf Inc.  Please state your name."
+	name = gets.chomp
+	# Recieve applicant stated age.
+	puts "Please state your age."
+	stated_age = gets.chomp.to_i
+	# Recieve applicant stated birthday
+	puts "Please state your year of birth."
+	birth_year = gets.chomp.to_i
+	birth_year_age = 2016 - birth_year
```

Comment:
with this setup you will have to change this file every year for it to work.

Check out the Ruby `Time` class. You should be able to handle a dynamic year check with something like `Time.now.year`